### ISSUE TITLE
模型为啥没收敛？

### ISSUE THREAD

**Issue by whtlkeep** [ProHiryu/bert-chinese-ner#24]
我clone下来数据，按原参数跑了跑，奇怪，没复原结果，不知道那里错了？
INFO:tensorflow:  eval_f = 0.6572645
INFO:tensorflow:  eval_precision = 0.7749974
INFO:tensorflow:  eval_recall = 0.57219213
INFO:tensorflow:  global_step = 4749
INFO:tensorflow:  loss = 60.91251

**Comment by zle1992** [comment]
100个epoch.....想想就可怕
